cli/generators/crud/index.js

```diff
@@ -7,6 +7,13 @@ module.exports = class extends Generator {
     super(args, opts);
 
     this.argument('name', { type: String, required: true });
+
+    this.option('api', {
+      desc: 'Does not include API related files',
```

Comment:
Crap. This commit is not supposed to be here. I didn't create a new branch. Will do it now.

Regarding your suggestion, yes, I like it better!